packages/@aws-cdk/aws-eks/lib/addons/alb-iam_policy-v2.0.0.json

```diff
@@ -0,0 +1,184 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
```

Comment:
Do you think it's worth it creating a test that goes through all these files and verifies that they are valid IAM policies? Just to prevent accidents or typos.

Reply:
Mm how would we do that though? Do we have a JSON iam policy validator?

Reply:
I haven't thought this through, but what about doing `iam.PolicyStatement.fromJson()` and check whether it throws an error?

Reply:
Added test 👍